### ISSUE TITLE
vue 3 support

### ISSUE THREAD

**Issue by jamesmsk** [JosephusPaye/vue-resize-detector#25]
will there be support for the new vue?

**Comment by JosephusPaye** [comment]
Yeah, I hope so. Have you tried running it with Vue 3? It'd be super useful to know what the errors are, if any.
